presto-hive/src/main/java/com/facebook/presto/twitter/hive/CachingHiveMetastoreRetryConfigurable.java

```diff
@@ -35,11 +35,15 @@
  * It added retry configuration and overrided the RetryDriver setup method
  * retry().
  * The class added the flexibility to set up retry configuration from a
- * runtime injector in order to solve "Failed to connect hive metastore" issue.
+ * runtime injector in order to solve "Failed connecting to hive metastore" issue,
+ * where the local cache of metastore failed to create a connection to remote metastore
+ * via HiveCluster.
  * All retry configurations are propergated from a
  * HiveClientRetryConfig instance.
  *
- * @see IQ-221
+ * This fix is a temporary fix and will keep investigating the issue.
+ *
+ * @see <a href="https://jira.twitter.biz/browse/IQ-221">IQ-221</a>
```

Comment:
Sorry, what I meant was please include a link to a jira who's sole purpose is the remove of this code, which is blocked on IQ-221, if this code is in fact temporary. That way once IQ-221 is resolved, we still have a clean-up jira. Also, best to include only the bug name withiout the full path to jira, since this will be visible externally.